packages/clerk-js/src/ui/router/PathRouter.tsx

```diff
@@ -34,26 +34,19 @@ export const PathRouter = ({ basePath, preservedParams, children }: PathRouterPr
     return window.location.search;
   };
 
+  const urlHasFragment = window.location.hash.startsWith('#/');
```

Comment:
This check exists in the helper as well, shall we refactor it or turn it into a tiny helper in url utils?

Reply:
sure